napalm/ios/ios.py

```diff
@@ -433,6 +435,13 @@ def commit_config(self):
         # Save config to startup (both replace and merge)
         output += self.device.send_command_expect("write mem")
 
+    def _canonical_int(self, interface):
```

Comment:
I think this one should be in the base driver.

Reply:
My thought process was

- Within the os, the only variable is interface
- Shorten from `napalm.base.helpers.canonical_interface(interface, "cisco_ios", True)` -> `self._canonical_int(interface)`